### PR TITLE
MW-228: Create Form

### PR DESCRIPTION
**Pull request by frlzjosh**

### CODE REVIEW

**Comment by alexhernandez** [comment]
WOW... commit city

**Comment by alexhernandez** [comment]
My Feedback is attached. See attachment.
[nav.zip](https://github.com/csun-metalab/meta-ui/files/1408432/nav.zip)

File: docs/form.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="../dist/css/meta-ui.css">
+    <link href="../node_modules/font-awesome/css/font-awesome.css" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="https://cdn.metalab.csun.edu/fonts/din/MyFontsWebfontsKit.css">
+    <title>Document</title>
+    <script src="../src/js/meta-ui.js"></script>
+</head>
+<body>
+<div class="section form">
```
Comment (jasmine1569) [inline docs/form.html]: this should have main--fullpage in it as well
example:

`<div class=" main--fullpage section`>

File: docs/form.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                    <li class="sidebar__item">
+                        <a href="../docs/type.html" class="sidebar__link">Typography</a>
+                    </li>
+                    <li class="sidebar__item active">
+                        <a href="../docs/form.html" class="sidebar__link">Form</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="col-md-9">
+                <p class="form__header">Adventurer Program</p>
```
Comment (jasmine1569) [inline docs/form.html]: this should be dark blue, also this has too much margin at the top

File: docs/form.html [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+                <input type="text" placeholder="Your First Name" class="form__input">
```
Comment (jasmine1569) [inline docs/form.html]: Your whole form text should be dark blue

File: src/sass/_navbar.scss [hunk 4]
```diff
@@ -0,0 +1,136 @@
+@import "variables";
```
Comment (jasmine1569) [inline src/sass/_navbar.scss]: why is the here in the _navbar.scss

File: src/sass/_base.scss [hunk 5]
```diff
@@ -1,3 +1,4 @@
+@import "variables";
```
Comment (jasmine1569) [inline src/sass/_base.scss]: this shouldn't be in the _base.scss

File: src/sass/_navbar.scss (hunk 4 above)
Comment (frlzjosh) [reply src/sass/_navbar.scss]: i think i did it

**Pull request merged**